tensorrt_llm/llmapi/llm_args.py

```diff
@@ -1714,30 +1689,70 @@ def validate_cuda_graph_config(self) -> 'TorchLlmArgs':
         2. If cuda_graph_batch_sizes is not provided, it is generated based on cuda_graph_max_batch_size
         3. If both are provided, cuda_graph_batch_sizes must match the generated values
         """
-        if self.cuda_graph_batch_sizes:
-            self.cuda_graph_batch_sizes = sorted(self.cuda_graph_batch_sizes)
-            if self.cuda_graph_max_batch_size != 0:
-                if self.cuda_graph_batch_sizes != self._generate_cuda_graph_batch_sizes(
-                        self.cuda_graph_max_batch_size,
-                        self.cuda_graph_padding_enabled):
-                    raise ValueError(
-                        "Please don't set both cuda_graph_batch_sizes "
-                        "and cuda_graph_max_batch_size.\n"
-                        f"cuda_graph_batch_sizes: {self.cuda_graph_batch_sizes}, "
-                        f"cuda_graph_max_batch_size: {self.cuda_graph_max_batch_size}"
-                    )
+        if self.cuda_graph_config is not None:
```

Comment:
A shorter alias could help the code more readable, e.g. `cg = self.cuda_graph_config`, and then the code will look as follows:

```python
        if cg is not None:
            if cg.cuda_graph_batch_sizes:
                cg.cuda_graph_batch_sizes = sorted(
                    cg.cuda_graph_batch_sizes)
                if cg.cuda_graph_max_batch_size != 0:
                    if cg.cuda_graph_batch_sizes != self._generate_cuda_graph_batch_sizes(
                            cg.cuda_graph_max_batch_size,
                            cg.cuda_graph_padding_enabled):
```